jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PushImageStep.java

```diff
@@ -37,8 +38,8 @@
 import java.util.concurrent.Callable;
 
 /**
- * Pushes a manifest for a tag. Returns the manifest digest ("image digest") and the container
- * configuration digest ("image id") as {#link BuildResult}.
+ * Pushes a manifestTemplate for a tag. Returns the manifestTemplate digest ("image digest") and the
+ * container configuration digest ("image id") as {#link BuildResult}.
```

Comment:
Hmm... I think we should do something about the "image id" when pushing a manifest list. It doesn't make sense to return a container configuration digest.

Reply:
A manifest list doesn't have a container config so yes we don't have an image id for a manifest list.

Reply:
`Pushes a manifest or a manifest list for a tag. If not a manifest list, returns the manifest digest ("image digest") and the container configuration digest ("image id") as {@link BuildResult}. If a manifest list, returns the manifest list digest only.`

And note the original Javadoc has a typo: `{#link BuildResult}` --> `{@link BuildResult}`